src/selectedtests/app/app.py

```diff
@@ -25,14 +34,30 @@ def create_app() -> Flask:
         doc="/swagger",
     )
 
+    # Creating index no-ops if index already exists
+    setup_indexes(mongo.test_mappings_queue())
```

Comment:
We might want to consider creating the indexes as a separate standalone command. The problem with creating them on application startup is that if we ever want to change the indexes, there has to be some coordination between deleting old indexes and restarting the app.